src/Select.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import styled from 'styled-components'
+import system, { space, fontSize } from 'styled-system'
+
+import theme from './theme'
+import Flex from './Flex'
+import Icon from './Icon'
+
+const SelectBase = styled.select`
+  appearance: none;
+  display: block;
```

Comment:
What does this block display try to ensure for the `select` style? When I toggled off the block display rule off in the story's inspector, the UI still looks okay. Trying to learn if this block display rule is trying to address certain use case.

Reply:
I think the `<select>` element defaults to `display: inline-block` – this is just to ensure that it's `display: block` though the `width: 100%` will probably make it behave similarly in the layout